obus.opam

```
@@ -14,7 +14,7 @@ build: [
 ]
 
 depends: [
-  "dune" {build}
+  "dune"
```

```suggestion
  "dune" {>= "1.1"}
```
According to the dune-project file, this package requires at least dune 1.1